packages/framework/presence/src/test/mockEphemeralRuntime.ts

```diff
@@ -5,13 +5,51 @@
 
 import { strict as assert } from "node:assert";
 
+import type { IAudience } from "@fluidframework/container-definitions";
 import type { ITelemetryBaseLogger } from "@fluidframework/core-interfaces";
-import type { IQuorumClients, ISequencedClient } from "@fluidframework/driver-definitions";
-import { MockQuorumClients } from "@fluidframework/test-runtime-utils/internal";
+import type {
+	IClient,
+	IQuorumClients,
+	ISequencedClient,
+} from "@fluidframework/driver-definitions";
+import { MockAudience, MockQuorumClients } from "@fluidframework/test-runtime-utils/internal";
 
 import type { ClientConnectionId } from "../baseTypes.js";
 import type { IEphemeralRuntime } from "../internalTypes.js";
 
+/**
+ * Creates a mock {@link @fluidframework/container-definitions#IAudience} for testing.
+ */
+export function makeMockAudience(clientIds: string[]): IAudience {
+	const clients = new Map<string, IClient>();
+	for (const [index, clientId] of clientIds.entries()) {
+		// eslint-disable-next-line unicorn/prefer-code-point
+		const stringId = String.fromCharCode(index + 65);
+		const name = stringId.repeat(10);
+		const userId = `${name}@microsoft.com`;
+		const email = userId;
+		const user = {
+			id: userId,
+			name,
+			email,
+		};
```

Comment:
How much of this is optional? I feel like email is certainly not guaranteed. To the extent that we can get away with the more horrible and useless data in audience, then we should. That way no production dependencies can be made on really clean data. If we stubbed things out with undefined or null that would be fine too since we don't want production dependencies on this stuff.